lib/gemika/matrix.rb

```diff
@@ -31,7 +36,13 @@ def initialize(attrs)
       # Returns whether this row can be run with the given Ruby version.
       #
       def compatible_with_ruby?(current_ruby = Env.ruby)
-        ruby == current_ruby
+        @used_ruby = aliased_ruby(ruby, current_ruby)
+
+        if @used_ruby
+          @used_ruby == current_ruby
+        else
+          ruby == current_ruby
+        end
```

Comment:
`aliased_ruby` always returns a ruby versions, so the `if` should always evaluate to `true` and the else path cannot be reached